### PR TITLE
feat: Revamp the UI of the Main Activity for better UI/UX

### LINKED ISSUE #117: Improve mainactivity UI
**Describe the chore**
The Main Activity UI can be made more user friendly, follow the screenshots that are attached for the implementation.

Home

Home_1

**Would you like to work on the issue?** (Yes/No)
No

### PR DESCRIPTION
**Pull request by ajay-prabhakar**
Fixed #117 

## Changes: 
Improved the Main activity according to mockup

## Screenshots of the change: 
ezgif com-video-to-gif (1)

### CODE REVIEW

**Review by abhijitramesh — CHANGES_REQUESTED** [review]
@Chromicle looks good, but please squash the commits.

**Review by abhijitramesh — APPROVED** [review]

**Review by yashk2000 — APPROVED** [review]
Great work only @Chromicle :smile:

**Comment by immadisairaj** [comment]
@Chromicle can you change the PR title to `improve` instead of `improved`. and the title tells `improved the main activity` can you specify what improvement is done.

**Comment by ajay-prabhakar** [comment]
> @Chromicle can you change the PR title to `improve` instead of `improved`. and the title tells `improved the main activity` can you specify what improvement is done.

yeah, done can you please review now

**Comment by immadisairaj** [comment]
> and the title tells `improved the main activity` can you specify what improvement is done.

??

File: app/src/main/java/org/amfoss/templeapp/ui/main/HomeActivity.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.SearchView;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import org.amfoss.templeapp.R;
+
+/** @author Chromicle. */
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/ui/main/HomeActivity.java:19]: I think we don't need these.

File: app/src/main/java/org/amfoss/templeapp/ui/main/PageViewModel.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package org.amfoss.templeapp.ui.main;
+
+import android.arch.core.util.Function;
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.Transformations;
+import android.arch.lifecycle.ViewModel;
+
+/** @author Chromicle. */
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/ui/main/PageViewModel.java:9]: also here

File: app/src/main/java/org/amfoss/templeapp/ui/main/PlaceholderFragment.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import org.amfoss.templeapp.R;
+
+/** @author Chromicle. */
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/ui/main/PlaceholderFragment.java:15]: same as above

File: app/src/main/java/org/amfoss/templeapp/ui/main/SectionsPagerAdapter.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import org.amfoss.templeapp.R;
+
+/** @author Chromicle. */
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/ui/main/SectionsPagerAdapter.java:11]: same as above. @Chromicle can you change this in all the files

File: app/src/main/java/org/amfoss/templeapp/ui/main/SectionsPagerAdapter.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Nullable
+    @Override
+    public CharSequence getPageTitle(int position) {
+        return mContext.getResources().getString(TAB_TITLES[position]);
+    }
+
+    @Override
+    public int getCount() {
+        // Show 3 total pages.
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/ui/main/SectionsPagerAdapter.java]: rephrase the comment

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -149,6 +149,23 @@
 
             </LinearLayout>
 
+
```
Comment (immadisairaj) [inline app/src/main/res/layout/activity_main.xml]: extra line

**Review by immadisairaj — CHANGES_REQUESTED** [review]

**Comment by yashk2000** [comment]
@immadisairaj we can leave out cutting the author annotations. Let them be. They were created by @Chromicle after all. 

@Chromicle let your author annotations be, but make the other changes as specified.

**Review by immadisairaj — APPROVED** [review]

**Pull request merged**